### PR TITLE
chore: fix grammar and typo

### PR DESCRIPTION
**Pull request by ldez**

### CODE REVIEW

File: docs/content/usage/cli/Options.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ To run the CLI without `sudo`, you have four options:
 
 ## Port Usage
 
-By default lego assumes it is able to bind to ports 80 and 443 to solve challenges.
+By default, lego assumes it is able to bind to ports 80 and 443 to solve challenges.
```
Comment (dmke) [inline docs/content/usage/cli/Options.md:37]: Do we have a canonical way to refer to this software? I've seen "lego", "Lego", and "LEGO".

This PR would be a great opportunity for make that consistent :)

File: e2e/fixtures/certs/README.md [hunk 2]
```diff
@@ -14,12 +14,11 @@ code!!! The CA's private key is **public** and anyone can use it to issue
 certificates that will be trusted by a system with the Pebble CA in the trust
 store.
 
-To re-create all of the Pebble certificates run:
+To re-create all the Pebble certificates run:
```
Comment (dmke) [inline e2e/fixtures/certs/README.md:17]:
```suggestion
To re-create all Pebble certificates run:
```

or

```suggestion
To re-create all certificates used by Pebble, run:
```

**Review by dmke — COMMENTED** [review]
(I'm half-way through, but something important came up; will continue later.)

File: docs/content/usage/cli/Options.md (hunk 1 above)
Comment (ldez) [reply docs/content/usage/cli/Options.md:37]: for me, it's "lego" or "LeGo" because it's not an acronym ("LEGO"), and "Lego" is a brick brand.

Comment (dmke) [reply docs/content/usage/cli/Options.md:37]: Wouldn't "Let's Encrypt in Go" lend itself to the backcronym "LEGo"? "Le go" is certainly funny :)

Personally, I prefer "Lego", although that danish toy producer might object...

> I'm not sure to want to handle this topic now :smile: 

Completely understandable.

**Review by dmke — CHANGES_REQUESTED** [review]
LGTM, two remarks though.

**Review by dmke — APPROVED** [review]

**Pull request merged**